tests/views/test_survey.py

```diff
@@ -78,6 +78,21 @@ def test_collection_exercise_view(self, mock_request):
         self.assertEqual(response.status_code, 200)
         self.assertIn("Business Register and Employment Survey".encode(), response.data)
         self.assertIn("000000".encode(), response.data)
+        self.assertIn("January 2017".encode(), response.data)
```

Comment:
Might be nicer in it's own tests so that if it fails because the events aren't there it's more obvious from the test name?